habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/util/HabushuUtil.java

```diff
@@ -82,50 +94,50 @@ public static void runBashScript(String bashScriptPath) {
      *                       INFO
      */
     public static void runBashScript(String bashScriptPath, String[] parameters, boolean debug) {
-	logger.debug("Running bash script located at {}.", bashScriptPath);
-
-	try {
-	    String[] command;
-	    if (parameters != null && parameters.length > 0) {
-		command = new String[parameters.length + 1];
-
-		for (int i = 0; i < parameters.length; i++) {
-		    command[i + 1] = parameters[i];
+		logger.debug("Running bash script located at {}.", bashScriptPath);
+
+		try {
+			String[] command;
```

Comment:
Looks like the indentation is off.  Can you reformat this file/

Reply:
Sorry I thought I had gotten all of the indentation sorted out, I gave it another pass just now.